tests/integration/settings.selectors.ts

```diff
@@ -0,0 +1,5 @@
+export enum SettingsSelectors {
+  NetworkListItem = 'network-list-item',
+  SettingsChangeNetwork = 'settings-change-network',
```

Comment:
`SettingsSelectors` with a child prefixed with "settings" seems redundant, it would be good to have the type of element we're trying to get, too.
```suggestion
  ChangeNetworkAction = 'settings-change-network',
```